### PR TITLE
DRILL-6044: Fixed shutdown button in Web UI when ssl,auth are enabled

### PR DESCRIPTION
**Pull request by dvjyothsna**
Fixed shutdown button to avoid cross domain requests since they are causing some issues when ssl and auth are enabled

### CODE REVIEW

File: exec/java-exec/src/main/resources/rest/index.ftl [hunk 1]
```diff
@@ -193,7 +192,11 @@
       var port = getPortNum();
       var timeout;
       var size = $("#size").html();
+      var host;
 
+      window.onload = function () {
+          host = location.host;
+      };
```
Comment (sohami) [inline exec/java-exec/src/main/resources/rest/index.ftl]: Please remove this `function()` and `var host`.

File: exec/java-exec/src/main/resources/rest/index.ftl [hunk 2]
```diff
@@ -269,9 +277,12 @@
...
       }
-      <#if model.shouldShowAdminInfo() || !model.isAuthEnabled() >
-          function shutdown(address,button) {
-              url = "http://"+address+":"+portNum+"/gracefulShutdown";
+       <#if model.shouldShowAdminInfo() || !model.isAuthEnabled()>
+          function shutdown(button) {
+              var protocol = location.protocol;
+              var host = location.host;
+              var requestPath = "/gracefulShutdown";
+              var url = protocol+host+requestPath;
```
Comment (sohami) [inline exec/java-exec/src/main/resources/rest/index.ftl]: How about refactoring above lines into separate function `getRequestUrl(requestPath)` and call that from both the methods (`shutdown` & `fillQueryCount`)

```
function getRequestUrl(requestPath) {
    var protocol = location.protocol;
    var host = location.host;
    var url = protocol + host + requestPath;
    return url;
}
```

**Comment by dvjyothsna** [comment]
@sohami Please review

**Comment by sohami** [comment]
+1 LGTM. Thanks for making the changes.

**Comment by parthchandra** [comment]
+1

**Pull request closed without merging**